lib/azure/armrest.rb

```diff
@@ -26,6 +26,7 @@ module Armrest
 require 'azure/armrest/storage_account_service'
 require 'azure/armrest/availability_set_service'
 require 'azure/armrest/disk_service'
+require 'azure/armrest/image_service'
```

Comment:
@bzwei The VirtualMachineImageService is not a resource group based service. It is designed to get publisher/offer/sku information. The ImageService (or, soon to be Storage::ImageService) is a resource group based service. It provides information for images that are currently used by VM's using managed disks.